lib/ApiResource.php

```diff
@@ -64,11 +64,10 @@ public static function classUrl()
     }
 
     /**
-     * @return string The full API URL for this API resource.
+     * @return string The instance endpoint URL for the given class.
      */
-    public function instanceUrl()
+    public static function classInstanceUrl($id)
```

Comment:
Sadly you can't have a static function and non-static function with the same name.